src/Faker/Provider/Company.php

```diff
@@ -27,4 +27,12 @@ public static function companySuffix()
     {
         return static::randomElement(static::$companySuffix);
     }
+
+    /*
+     * @example 'Cashier'
+     */
+    public static function jobTitle()
+    {
+        return static::randomElement(static::$jobTitle);
```

Comment:
as `$jobTitle` is undefined in the Base provider, this will throw an error in all but one locale. Please add a default value for the base locale, for instance a random word (using the Lorem provider).